Apps/phcofensetriage/readme.html

```diff
@@ -2,135 +2,177 @@
+      <th>Description</th>
+      <th>Notes</th>
+    </tr>
+    <tr>
+    <tr>
+      <td>report_type</td>
+      <td>Type of reports to retrieve, default retrieve All.</td>
+      <td>Possible values are; All - All reports in Inbox, Recon, and Processed folders; Inbox - Uncategorized reports
+        in the Inbox and Recon folders; Processed - Categorized reports in the Processed folder.
+        Be aware that the reports in the Recon folder are ingested only if the option is All or Inbox but not Processed.
+        Reports from the Inbox folder are unreviewed reports and therefore missing any evaluated information</td>
+    </tr>
     <td>report_ingest_subfields</td>
-    <td>Only applicable if ingesting reports. This option will ingest the dictionary and list fields of the subject as additional artifacts.
-</td>
-    <td>If set to true, during ingestion of reports, in addition to the Report Artifact which contains the entire report as an artifact, it will extract various sub-elements and create individual artifacts for the following items; URLs, tags, rules, and attachments.
-  </tr>
-  <tr>
-    <td>report_match_priority</td>
-    <td>The highest match priority based on rule hits for the report.
-</td>
-    <td></td>
-  </tr>
-  <tr>
-    <td>report_category_id</td>
-    <td>Filter by category ID, default retrieve All.</td>
-<td>The category ID (1-5) for processed reports. Takes either string or number. Only valid when retrieving "All" or "Processed" reports. Category IDs correspond to category names as follows: 5 (lowest): Phishing Simulation; 1: Non-Malicious; 2: Spam; 3: Crimeware; 4 (highest): Advanced Threats.
- You may retrieve all results or filter results by a single category. At the moment, if you wish to ingest two categories, it is suggested you create a second asset.
-</td>
-  </tr>
-<tr>
-    <td>report_tags</td>
-    <td>One or more tags of processed reports to filter on. Use commas to separate multiple tags.
-</td>
-    <td></td>
-  </tr>
-</table>
-<p>
+    <td>Only applicable if ingesting reports. This option will ingest the dictionary and list fields of the subject as
+      additional artifacts.
+    </td>
+    <td>If set to true, during ingestion of reports, in addition to the Report Artifact which contains the entire report
+      as an artifact, it will extract various sub-elements and create individual artifacts for the following items;
+      URLs, tags, rules, and attachments.
+      </tr>
+      <tr>
+        <td>report_match_priority</td>
+        <td>The highest match priority is based on rule hits for the report.
+        </td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>report_category_id</td>
+        <td>Filter by category ID, default retrieve All.</td>
+        <td>The category ID (1-5) for processed reports. Takes either string or number. Only valid when retrieving "All"
+          or "Processed" reports. Category IDs correspond to category names as follows: 5 (lowest): Phishing Simulation;
```

Comment:
These category numbers are confusing - 5 is the lowest and 4 is the highest?  Is this specific to the app or to Cofense Triage?